Documentation/network/l2-announcements.rst

```diff
@@ -0,0 +1,314 @@
+node to claim a lease gets it. This might cause asymmetric traffic distribution.
+
+Leases
+------
+
+The leases are created in the same namespace where Cilium is deployed, 
+typically ``kube-system``. You can inspect the leases with the following command:
+
+.. code-block:: shell-session
+
+    $ kubectl -n kube-system get lease
+    NAME                                  HOLDER                                                    AGE
+    cilium-l2announce-default-deathstar   worker-node                                               2d20h
+    cilium-operator-resource-lock         worker-node2-tPDVulKoRK                                   2d20h
+    kube-controller-manager               control-plane-node_9bd97f6c-cd0c-4565-8486-e718deb310e4   2d21h
+    kube-scheduler                        control-plane-node_2c490643-dd95-4f73-8862-139afe771ffd   2d21h
+
+The leases starting with ``cilium-l2announce-`` are leases used by this feature.
+The last part of the name is the namespace and service name. The holder indicates
+the name of the node that currently holds the lease and thus announced the IPs 
+of that given service.
+
+To inspect a lease:
+
+.. code-block:: shell-session
+
+    $ kubectl -n kube-system get lease/cilium-l2announce-default-deathstar -o yaml
+    apiVersion: coordination.k8s.io/v1
+    kind: Lease
+    metadata:
+      creationTimestamp: "2023-05-09T15:13:32Z"
+      name: cilium-l2announce-default-deathstar
+      namespace: kube-system
+      resourceVersion: "449966"
+      uid: e3c9c020-6e24-4c5c-9df9-d0c50f6c4cec
+    spec:
+      acquireTime: "2023-05-09T15:14:20.108431Z"
+      holderIdentity: worker-node
+      leaseDurationSeconds: 3
+      leaseTransitions: 1
+      renewTime: "2023-05-12T12:15:26.773020Z"
+
+The ``acquireTime`` is the time at which the current leader acquired the lease.
+The ``holderIdentity`` is the name of the current holder/leader node. 
+If the leader does not renew the lease for ``leaseDurationSeconds`` seconds a
+new leader is chosen. ``leaseTransitions`` indicates how often the lease changed
+hands and ``renewTime`` the last time the leader renewed the lease.
+
+There are three Helm options that can be tuned with regards to leases:
+
+* ``l2announcements.leaseDuration`` determines the ``leaseDurationSeconds`` value
+  of created leases and by extent how long a leader must be "down" before 
+  failover occurs. (default is 15s)
+
+* ``l2announcements.leaseRenewDeadline`` is the interval at which the leader 
+  should renew the lease. (default is 5s)
+
+* ``l2announcements.leaseRetryPeriod`` if renewing the lease fails, how long 
+  should the agent wait before it tries again. (default is 2s)
+
+.. tabs::
+    .. group-tab:: Helm
+
+        .. parsed-literal::
+
+            $ helm upgrade cilium |CHART_RELEASE| \\
+               --namespace kube-system \\
+               --reuse-values \\
+               --set l2announcements.enabled=true \\
+               --set kubeProxyReplacement=strict \\
```

Comment:
Same as above. This is basically a non-functional `helm` example. Should we add the following arguments like we do in the [KPR guide](https://docs.cilium.io/en/stable/network/kubernetes/kubeproxy-free/#quick-start)?
```
    --set k8sServiceHost=${API_SERVER_IP} \
    --set k8sServicePort=${API_SERVER_PORT}
```